src/main/webapp/pages/accessionvalidationrange/accessionValidationRange.jsp

```diff
@@ -104,11 +114,36 @@ function /*void*/ enableDisableCheckboxes( matchedElement, groupingNumber ){
 	jQuery("selectAllAccept").checked = false;
 }
 
-function criticalResultCheck( index ){
-	var result = jQuery("#resultId_" + index).val();
-	if((result >= 0) && (result <= 10) || (result >= 95)){
-		alert( `<%=MessageUtil.getContextualMessage("result.critical")%>` );
-	} 
+function criticalResultCheckLowRangeHigh( index ){
```

Comment:
This checking if a result is in a critical range will check to see if the value for this test is a critical range for ANY test, not just the one it is configured for (unless I am missing something). This logic seems like it should go beside normal, and reporting range validation inside testResults.js validateResults(). 

Values can be collected and embedded in the page via TestResultItem.java in the same way that lowerAbnormalRange is via ResultLoadUtility.setResultLimitDependencies. Search resultListView.jsp for <c:set var="lowerBound"  to see how these values are included on the page.